src/app/types/bounty/Bounty.ts

```diff
@@ -1,5 +1,6 @@
 import { Double, Int32, ObjectId } from 'mongodb';
 
+// TODO - *TWE I don't think we need both this and BountyCollection. Settle on one or the other
```

Comment:
yea, very much agree with this. 

i'm in favor of keeping `BountyCollection`. The history of `Bounty` was to strongly type the object that was first inserted into the Db as part of `Create` (thus the optional and non-optional params), but given we type check any documents we retrieve from the db, that's no longer necessary

Reply:
I tried in the create action, but it causes type checking issues. Need a plan to change that I'd like to keep out of scope for now.